### PR TITLE
pull request for demo

### PR DESCRIPTION
**Pull request by sonamoo**
i changed the color of the heart counter

### CODE REVIEW

File: src/components/CompanyList/presenter.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ const styles = {
     flex: 1,
   },
   HeaderContentsText: {
-    color: 'green',
+    color: 'red',
```
Comment (sonamoo) [inline src/components/CompanyList/presenter.js]: this looks good

Comment (junlee91) [reply src/components/CompanyList/presenter.js]: talking to yourself...? 😳

File: android/app/build.gradle [hunk 2]
```diff
@@ -149,6 +149,8 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-vector-icons')
+    compile project(':react-native-splash-screen')
```
Comment (junlee91) [inline android/app/build.gradle]: This will break a build in Android!!

File: android/app/src/main/java/com/careertalk/MainApplication.java [hunk 3]
```diff
@@ -25,6 +27,8 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
+            new VectorIconsPackage(),
+            new SplashScreenReactPackage(),
```
Comment (junlee91) [inline android/app/src/main/java/com/careertalk/MainApplication.java]: This part will throw an error!!

Comment (junlee91) [reply android/app/src/main/java/com/careertalk/MainApplication.java]: Because of duplication

**Pull request merged**